playbooks/byo/openshift-cluster/upgrades/v3_4/upgrade_control_plane.yml

```diff
@@ -94,5 +94,7 @@
   - include: ../../../../common/openshift-cluster/upgrades/cleanup_unused_images.yml
 
 - include: ../../../../common/openshift-cluster/upgrades/upgrade_control_plane.yml
+  vars:
+    master_config_hook: "v3_4/master_config_upgrade.yml"
```

Comment:
This needs to be also be set when we include upgrade_control_plane.yml in the regular upgrade.yml, same dir as this file.